### PR TITLE
Next gen gh pages

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Update Next gen gh-pages.
- Bump Lib.
- Bump Docs.

### CODE REVIEW

**Comment by juancarlospaco** [comment]
@joliveros Please, when you can Review this, thanks...

File: package.json [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
   "engines" : { "node" : ">=4.1.0 <5.5.0" },
   "dependencies": {
-    "body-parser": "^1.6.5",
-    "express": "^4.8.5",
-    "twilio": "^1.6.0",
-    "morgan": "^1.2.3",
-    "jade": "1.7.x"
+    "body-parser": "^1.15.2",
+    "express": "^4.14.0",
+    "twilio": ">=3.0.0-rc.13",
```
Comment (joliveros) [inline package.json]: deberia ser `3.0.0-rc.13` en vez de `>=3.0.0-rc.13"`

**Comment by joliveros** [comment]
Looks good to me. Only question for @acamino is since `master` was ahead of `gh-pages` when @juancarlospaco rebased `next-gen` unto `gh-pages-next-gen` he brought along all of the commits that were ahead in master. Is that acceptable?

**Comment by acamino** [comment]
This branch includes tons of commits that actually doesn't belong here.

**Pull request closed without merging**